_data/student_projects.yml

```diff
@@ -25,3 +25,8 @@
   github: https://atomickindness.tk/
   author: Ploypiti P.
   image: AAK_logo.png
+
+- name: Aditya Dutt
+  github: duttaditya18/beautiful-flask
```

Comment:
Can you add `https://github/` for the link.

Reply:
Done!